src/Resources/translations/SonataAdminBundle.it.xliff

```diff
@@ -476,15 +476,15 @@
       </trans-unit>
       <trans-unit id="read_more">
         <source>read_more</source>
-        <target>Leggi di più</target>
+        <target>Continua a leggere</target>
```

Comment:
I'm not a Italian speaker, but "Read more" is not exactly the same as "Keep reading".
The English translation says "Read more"
https://github.com/sonata-project/SonataAdminBundle/blob/201b331387c3eaec1db94cd6a54b4098acc1606f/src/Resources/translations/SonataAdminBundle.en.xliff#L479

Reply:
Hello @phansys and **thank you** very much for taking the time to check this out. Actually, **you are right**! The literal translation (like a machine would) is different, however when in the sense of _continue reading_ a block of text (like an article or blog post) we prefer this other expression. "_Leggi di più_" is more commonly used in Italian in the context of "_Read more books and educate yourself_". Indeed, the original expression wasn't wrong, but it was a word per word translation, whereas mine is more of a localization: just cosmetic but as a native Italian speaker (although **not a professional translator**) felt more at home.